content/en/tracing/legacy_app_analytics/_index.md

```diff
@@ -19,12 +19,11 @@ To migrate to the new configuration options, remove all legacy App Analytics [co
 
 ## App Analytics setup
 
-To enable App Analytics, either [automatically](#automatic-configuration) or [manually](#custom-instrumentation), configure your APM tracer to emit the relevant analytics from your services.
+App Analytics configuration options are located in the tracing libraries and in the Datadog Agent. In the libraries, analytics spans from your services are generated either [automatically](#automatic-configuration) or [manually](#custom-instrumentation).
 
+### In Tracing Libraries
```

Comment:
```suggestion
### In tracing libraries
```

Reply:
Not sure if 'tracing libraries' is meant to be a proper noun, could you confirm?

Reply:
Let's keep the capital case, we do that in other places :)